src/config.c

```diff
@@ -2544,18 +2537,18 @@ standardConfig configs[] = {
     createBoolConfig("aof-load-truncated", NULL, MODIFIABLE_CONFIG, server.aof_load_truncated, 1, NULL, NULL),
     createBoolConfig("aof-use-rdb-preamble", NULL, MODIFIABLE_CONFIG, server.aof_use_rdb_preamble, 1, NULL, NULL),
     createBoolConfig("aof-timestamp-enabled", NULL, MODIFIABLE_CONFIG, server.aof_timestamp_enabled, 0, NULL, NULL),
-    createBoolConfig("cluster-replica-no-failover", "cluster-slave-no-failover", MODIFIABLE_CONFIG, server.cluster_slave_no_failover, 0, NULL, updateClusterFlags), /* Failover by default. */
```

Comment:
i'd rather avoid these renames.
changes in this block have a high potential to cause unnecessary merge conflicts since each of these configs is just one line, so if an adjacent line is also changed it causes a useless conflict.

also, even if in some PR / branch / release / fork, there are some other change to this line (like flags or alias), there's no need to cause cause a conflict.

i think we can keep the terms "update" and "apply" mixed.

p.s. i'm ok to change the tls block (no need to create / define apply aliases),
i presume this block won't cause much conflicts

Reply:
ok